funnel/assets/js/scan_badge.js

```diff
@@ -34,61 +34,69 @@ const badgeScan = {
         this.set('showModal', false);
         this.startRenderFrameLoop();
       },
-
-      checkinAttendee(qrcode) {
+      async checkinAttendee(qrcode) {
         this.set({
           scanning: true,
           showModal: true,
         });
         $('#status-msg').modal('show');
-        const url = checkinApiUrl.replace('puk', qrcode.substring(0, 8));
+        const url = `${checkinApiUrl.replace('puk', qrcode.substring(0, 8))}?t`;
```

Comment:
What is the `?t` suffix for?

Reply:
Checkin set to true

Reply:
We should be using the thorough regex used here: https://github.com/taoeffect/vue-script2#writing-script-instead-of-script2-using-script2ify